src/app/modules/main/profile_section/contacts/module.nim

```diff
@@ -203,6 +203,9 @@ method contactNicknameChanged*(self: Module, publicKey: string) =
 method contactTrustStatusChanged*(self: Module, publicKey: string, trustStatus: TrustStatus) =
   self.view.contactsModel().updateTrustStatus(publicKey, trustStatus)
 
+method onTrustStatusRemoved(self: Module, publicKey: string) =
+  self.view.emitTrustStatusRemoved(publicKey)
```

Comment:
You can call `self.view.trustStatusRemoved(publicKey)` from here directly if you want

Reply:
Done